ember-cli-build.js

```diff
@@ -110,14 +107,70 @@ module.exports = function() {
     rollupPackage(packagesES, '@ember/-internals/container'),
   ]);
 
+  let bundlesES5 = toTranspiledBundles(
```

Comment:
Thoughts on swapping things around a bit such that this is a straight pipeline (doing the es bundle then doing another babel pass on it to add the extra processing)?